src/ert/run_models/base_run_model.py

```diff
@@ -378,11 +378,21 @@ def run_ensemble_evaluator(
     ) -> List[int]:
         ensemble = self._build_ensemble(run_context)
 
-        successful_realizations = EnsembleEvaluator(
-            ensemble,
-            ee_config,
-            run_context.iteration,
-        ).run_and_get_successful_realizations()
+        if FeatureScheduler.is_enabled(self.queue_system):
+            asyncio.set_event_loop(new_event_loop())
+            successful_realizations = get_event_loop().run_until_complete(
+                EnsembleEvaluatorAsync(
+                    ensemble,
+                    ee_config,
+                    run_context.iteration,
+                ).run_and_get_successful_realizations()
+            )
```

Comment:
Do we have to set the event loop here?
ChatGPT says we should avoid altering the event loop in the current thread, and suggests doing this instead:
```
# Create a new event loop
loop = asyncio.new_event_loop()

# Run the coroutine to completion using the new event loop
successful_realizations = loop.run_until_complete(
    EnsembleEvaluatorAsync(
        ensemble,
        ee_config,
        run_context.iteration,
    ).run_and_get_successful_realizations()
)

# Close the loop after use
loop.close()
```

Reply:
You might be right. I'll give it a try.

Reply:
Is this what did not work?